src/components/common/YourCompany/BusinessContactInfo.vue

```diff
@@ -84,14 +82,7 @@ export default class BusinessContactInfo extends Vue {
     if (isEqual(contactInfo, this.getBusinessContact)) return
 
     try {
-      if (
-        this.isAlterationFiling ||
-        this.isCorrectionFiling ||
-        this.isFirmChangeFiling ||
-        this.isSpecialResolutionFiling
```

Comment:
I found this while testing -- it didn't work for restorations.

With the new Your Company Wrapper, we either include this component or we don't, so we no longer need to check here whether it's the allowable filing type.